src/components/Lpcontent/Lpcontent.css

```diff
@@ -0,0 +1,13 @@
+.top-service-card{
```

Comment:
CSS file name should be changed to Topservicecard.css as the css file is styling the component Topserviceard